### PR TITLE
add folder-structure

### PR DESCRIPTION
**Pull request by SelcoD**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="/css/styles.css" />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>quizz app</title>
```
Comment (josch87) [inline index.html:8]: I found a typo. Or is this your apps individual name?

**Review by josch87 — APPROVED** [review]
Looks good. Maybe there is a type, unsure if you wanted to give it an individual name ;)

File: index.html (hunk 1 above)
Comment (SelcoD) [reply index.html:8]: what do you mean ? the title ? quiz app is the title !

Comment (SelcoD) [reply index.html:8]: ahhh you mean quiZZ...let change it

**Review by VolkerJacobsen — COMMENTED** [review]
Sieht doch gut aus! ;)

**Pull request merged**